src/test/test_connector_http.cpp

```diff
@@ -21,21 +21,33 @@ using namespace jsonrpc;
 using namespace std;
 
 #define TEST_PORT 8383
-#define CLIENT_URL "http://localhost:8383"
+#define CLIENT_URL "http://127.0.0.1:8383"
+
+/* When the server is listening on all interfaces, then also this alternate
+   URL should be fine to connect to it.  When it is bound specifically to
+   127.0.0.1, though, then this should not work.  */
+#define ALTERNATE_CLIENT_URL "http://127.0.0.2:8383"
```

Comment:
`CLIENT_URL` and `ALTERNATE_CLIENT_URL` are the same?

Reply:
No they are not - the alternate IP is 127.0.0.2 rather than 127.0.0.1.  This allows us to test the bind address feature - if binding explicitly to 127.0.0.1, then a connection to 127.0.0.2 fails.  If binding to all interfaces, then both addresses should work.  The unit tests verify that.

Reply:
I didn't spot the `2`. You are right.